src/features/flows/pairingConfig.ts

```diff
@@ -92,7 +92,7 @@ const googleHomeGuideStartBase = [
     'Once device is connected, tap **Done**.',
 ];
 
-const googleHomeGuideEndBase = ['The device is visible in the selected room.'];
+const googleHomeGuideEndBase = [''];
```

Comment:
I don't know if that is the cause, but for pairing light bulb with Google, there is an empty 13 step 
<img width="588" height="151" alt="image" src="https://github.com/user-attachments/assets/9fb02ef7-af76-4c78-9cdd-e29a7e05607d" />